web/themes/custom/novel/templates/layout/menu--main.html.twig

```diff
@@ -0,0 +1,11 @@
+
+<ul class="header__menu-navigation">
+{% for item in items %}
+  <li class="header__menu-navigation-item">
+    <a href="{{ item.url }}" class="header__menu-navigation-link text-body-medium-regular hide-linkstyle">
+      {{ item.title }}
+    </a>
```

Comment:
Consider using the [`link()` function](https://www.drupal.org/docs/develop/theming-drupal/twig-in-drupal/functions-in-twig-templates#s-linktext-uri-attributes) and `item.attributes`.

Attributes is often helpful if we get to marking the current menu item as active. The link function also has helpers specifically regarding menu items that may be helpful.

Reply:
Good point, done 👍